content/docs/concepts/config.md

```diff
@@ -573,7 +575,7 @@ There are three ways to configure providers:
 
 Please note:
 
-* Configuration file settings are only used by the default provider. If you instantiate a provider object, it will not read values from the stack configuration.
+* Configuration file settings are only used by the default provider. If you instantiate a custom provider object, it will not read values from the stack configuration.
 * The precedence of configuration sources (configuration file, environment and args) can vary between providers. Please refer to the provider's documentation for specific configuration instructions.
 
 ## Pulumi Configuration Options
```

Comment:
@cnunciato Regarding this section I have the following questions:

- What does disabling default providers do? What are use cases in which this functionality is useful?
- ”Pulumi CLI only creates or updates tags which are listed in the config. If you remove a tag from the stack config, you have to remove it from the stack in Pulumi Cloud manually.” Do you happen to know why?